contrib/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# example of Dockerfile that builds release of electrumx-1.13.0
+# ENV variables can be overrided on the `docker run` command
+
+FROM ubuntu:18.04
+
+WORKDIR /
+ADD https://github.com/kyuupichan/electrumx/archive/1.13.0.tar.gz /
+RUN tar zxvf *.tar.gz
+
+RUN apt-get update && \
+        apt-get -y install python3.7 python3-pip librocksdb-dev libsnappy-dev libbz2-dev libz-dev liblz4-dev && \
+        pip3 install aiohttp pylru python-rocksdb
+
+RUN cd /electrumx* && python3 setup.py install
+
+ENV SERVICES="tcp://:50001"
+ENV COIN=BitcoinSV
+ENV DB_DIRECTORY=/db
+ENV DAEMON_URL="http://username:password@hostname:port/"
```

Comment:
lmao where is the bitcoind?

Reply:
ENV can also be set at `docker run` command to override anything to their needs.